profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
```

Comment:
Yep, agreed in general I do think there is some unfortunate redundancy in 90% of our packages. However there are two main reasons that I structure the packages this way:
1. Readability: If the last element of the path here were "2017-04-19" that would mean folks would have to reference the package as "2017" all throughout their code; that approach is clearly ridiculous.
2. Not all packages have this redundancy. Compute for example has two sets of packages: `compute` & `containerService`. To disambiguate in these cases and have consistent structure, we must have some redundancy.